lib/utils.js

```diff
@@ -101,7 +101,9 @@ var retry = function retry(f, options) {
       // Compute delay
       var delay = Math.pow(2, retry) * options.delayFactor;
       var rf = options.randomizationFactor;
-      delay = delay * (Math.random() * 2 * rf + 1 - rf);
+      if (rf) {
+        delay = delay * (Math.random() * 2 * rf + 1 - rf);
```

Comment:
I wonder why this math is like it is in particular.

Reply:
I think it aims to get a uniform distribution between (1 + rf) and (1 - rf).
* random returns 0 => 0 * 2 * rf + 1 - rf => 1 - rf
* random returns 1 => 1 * 2 * rf + 1 - rf => 1 + rf